cdap-ui/app/cdap/components/Wrangler/index.js

```diff
@@ -53,6 +53,10 @@ export default class Wrangler extends Component {
     this.onWrangleClick = this.onWrangleClick.bind(this);
   }
 
+  // componentDidMount() {
```

Comment:
No, this is for development purpose, so that I can skip the first screen of copy paste input